spyder/app/mainwindow.py

```diff
@@ -3422,6 +3426,20 @@ def _check_updates_ready(self):
         # Provide feeback when clicking menu if check on startup is on
         self.give_updates_feedback = True
 
+    @Slot()
+    def show_tour_message(self):
+        """
+        Show message about starting the tour the first time Spyder starts.
+        """
+        dev_mode = os.environ.get('SPYDER_DEV')
+        if not running_under_pytest() and not dev_mode:
+            show_tour = CONF.get('main', 'show_tour_message')
+            if show_tour:
+                CONF.set('main', 'show_tour_message', False)
+                self.tour_dialog = tour.OpenTourDialog(
+                    self, lambda: self.show_tour(0))

```

Comment:
It seems my suggestion before got lost—instead of hardcoding this magic number here, I suggest adding a module-level constant `DEFAULT_TOUR = 0` at the top of mainwindow.py or tour.py (@ccordoba12 opinion which?) and using that here and in the function that builds the tour menu that was added in #13717 (this will need a rebase).